tests/translator/input/all_policy_templates.yaml

```diff
@@ -98,6 +98,9 @@ Resources:
 
         - CloudWatchDashboardPolicy: {}
 
+        - RekognitionFacesManagementPolicy:
```

Comment:
Adding this policy to the end of the list instead of the middle should fix the failing tests.